doc/src/modules/physics/continuum_mechanics/beam_problems.rst

```diff
@@ -44,6 +44,13 @@ Examples
 ========
 
 Below are examples of a variety two dimensional beam bending problems.
+ 
+.. note::
+
+    While solving a beam bending problem, a user is free to choose its
+    own sign convention as long as the upward forces and clockwise moments
+    have the same signs. The results will
+    automatically follow the chosen sign convention.
```

Comment:
I don't find this obviously informative.

Some things that would be helpful to know:

- Is a specific sign convention required for any aspects of the Beam functionality (construction, solving, etc.)?
- If so, what conventions are assumed and where?
- Can a user use any sign convention they desire as long as they are consistent? If so, what is required to be consistent?

The answers to these questions could help provide insight into what the best message should be.

Also, an explanation of the sign convention belongs most importantly in the docstrign of `Beam()`.